psst-gui/src/main.rs

```diff
@@ -33,12 +33,14 @@ fn main() {
     .init();
 
     let config = Config::load().unwrap_or_default();
+    let username = config.get_username();
```

Comment:
This won't really work the first time user launches the app. Can we maybe inject the username (or the `LocalTrackManager`) into `WebApi` a bit later?